### DISCUSSION TITLE
How to train

### DISCUSSION THREAD

**Discussion by daniel-scalioni** [urchade/GLiNER#126] [General]
Hi urchade, first of all, congrats for this work. It's awsome.
But I'm trying to reproduce your paper training, but I had some issues.
I'm using train_instruct.json as database.
And I tried to use booth finetune.ipynb and train.py, but on each of them loss drops too fast and there is no result when I try to inference.
Could you, please, inform which version can I use to reproduce your paper data?
Or one version that you know that it is working fine?
Thanks for support.

I'm using version 0_2_2 but earlier I tryed on anon old version.
By the way I'm trying to funetune a Portuguese BERT from scratch to improuve GLiNER on PT/BR.

**Comment by urchade** [comment]
For now, you can refer to this repo: https://github.com/urchade/GLiNER/tree/flair_version, and train using `config_span.yaml` to reproduce the results from the paper

``python train.py --config config_span.yaml``

**Reply by Rishik00** [reply]
Heyy, I was trying to run this in my colab environment but seem to be facing CUDA out of memory errors. What are the minimum requirements for replicating your results?
